apps/redi-talent-pool/src/pages/front/signup/SignUp.tsx

```diff
@@ -50,12 +49,12 @@ const formCourses = coursesWithAlumniDeduped.map((course) => {
   }
 })
 
-const howDidHearAboutRediOptions = HOW_DID_HEAR_ABOUT_REDI_OPTIONS.map(
-  (option) => ({
-    value: option,
-    label: option,
-  })
-)
+const howDidHearAboutRediOptionsMap = Object.keys(
```

Comment:
Works, but feel free to bring it more in line with the technique used elsewhere that relies on `Object.entries`. Besides code pattern homogeneity, I also think it's cleaner since the function you pass to `map` becomes pure.